### DISCUSSION TITLE
Options to set a default zoom value

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4046] [Ideas]
I apologize if I overlooked this, but I was wondering if anyone would be interested in adding the ability to set a default zoom value as a setting.  This would allow Geany to launch already zoomed to the value set.

Thank You

**Comment by ntrel** [comment]
Would that be advantageous over the existing ability to set a different editor font size?

**Comment by advice2020** [comment]
Thank you for your response.
I'm glad you asked :)
This would allow the font to remain at a standard size but it would make use of the zoom in/out feature.
For example, this way if user ran into a situation where the font was maybe too large to fit the current document/Geany windows width, they could then easily make use of the Ctrl+mouse scroll wheel to make adjustments on the fly, without having to keep going into the dialog box to change the font size.  Being that Geany does not allow the use of mulitple size fonts, zoom feature seems more flexible for certain scenarios.  Would be nice for user to be able to set the default zoom to the size they use most often.

**Reply by eht16** [reply]
> Thank you for your response. I'm glad you asked :) This would allow the font to remain at a standard size but it would make use of the zoom in/out feature. 

> For example, this way if user ran into a situation where the font was maybe too large to fit the current document/Geany windows width, they could then easily make use of the Ctrl+mouse scroll wheel to make adjustments on the fly, without having to keep going into the dialog box to change the font size. 

This is currently already possible or I didn't get it.

> Being that Geany does not allow the use of mulitple size fonts

How would "multiple size fonts" work?

I still don't understand the use case at all.
